conan/api/subapi/cache.py

```diff
@@ -63,6 +63,8 @@ def package_path(self, pref: PkgReference):
         app = ConanApp(self.conan_api)
         pref = _resolve_latest_pref(app, pref)
         ref_layout = app.cache.pkg_layout(pref)
+        if os.path.exists(ref_layout.install()):
```

Comment:
So that calling `conan cache path name/ver:12345` for a package that has an install method correctly returns the `install` folder instead of the original package folder, which is in line with the transparent replacement done elsewhere

Reply:
Note that `conan download` does not run the install method, it's only ran the first time it's needed as part of a graph info, so until then `conan cache path` does return the original pacvkage folder even if an `install()` method exists